tslint.json

```diff
@@ -19,7 +19,6 @@
         "no-console": [true],
         "no-empty-interface": false,
         "no-trailing-whitespace": [true],
-        "no-unused-variable": [true],
```

Comment:
This shouldn't be removed, use underscores in the code if an unused variable appears.